frontend/src/App.tsx

```diff
@@ -1,21 +1,38 @@
 import { BrowserRouter, Route, Routes } from "react-router-dom";
-import Handbook from "./Handbook";
+import Catalog from "./Catalog";
 import Leader from "./Leader";
 import LeadersList from "./LeadersList";
 import Login from "./Login";
+import NavigationMenu from "./NavigationMenu";
 import Project from "./Project";
 import ProjectsList from "./ProjectsList";
 import Settings from "./Settings";
 import Statistics from "./Statistics";
 import Tags from "./Tags";
+import { UserType } from "./types/global.d";
 import Users from "./Users";
+import { Box } from "@mui/material";
+import { Stack } from "@mui/system";
+import React from "react";
 
 function App() {
+
+  const [user, setUser] = React.useState(UserType.Editor)
```

Comment:
I would set `Admin` here. For example I got confused why I don't see the users management page.

Reply:
Ok, but it's just for testing. I wrote about it in the comment in next line